tests/php/ORM/DataObjectSchemaTest.php

```diff
@@ -373,4 +399,76 @@ public function testWriteUniqueIndexes()
             AllIndexes::get()
         );
     }
+
+    public function provideHasOneComponent()
```

Comment:
```suggestion
    public function provideHasOneComponent(): array
```
Move provide method below the test method

Reply:
Done